src/common/options/rgw.yaml.in

```diff
@@ -3692,3 +3692,23 @@ options:
   default: tank
   services:
   - rgw
+- name: rgw_d4n_directory_host
+  type: str
+  level: advanced
+  desc: The rgw directory host
+  default: 127.0.0.1
```

Comment:
Have we considered the case where there are multiple RGWs running this with multiple Redises. Is that the next step after this PR is merged? If that's the case we'd probably want multiple endpoint values in one config variable.

Reply:
Yes, that is definitely a future step because currently, this code can only support one redis instance.